controllers/alert_controller.go

```diff
@@ -52,16 +54,16 @@ func (r *AlertReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 
 	init := true
 	for _, condition := range alert.Status.Conditions {
-		if condition.Type == v1alpha1.ReadyCondition && condition.Status == corev1.ConditionTrue {
+		if condition.Type == meta.ReadyCondition && condition.Status == corev1.ConditionTrue {
 			init = false
 			break
 		}
 	}
```

Comment:
This can now theoretically be replaced by something like

```suggestion
	if c := meta.GetCondition(alert.Status.Conditions, meta.ReadyCondition); c != nil {
		if c.Status == corev1.ConditionTrue {
			init = false
		}
	}
```